go.mod

```diff
@@ -1,16 +1,18 @@
 module github.com/datadog/orchestrion
 
-go 1.21
+go 1.22.0
```

Comment:
I thought you did not wanted to switch to 1.22 because of Go Support Policy

Reply:
😡 this is `go mod tidy` doing things nobody ever asked it to 😡